### PR TITLE
repo security improvements

### PR DESCRIPTION
**Pull request by paOol**
integrate njsscan and add dependabot

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  # Enable version updates for npm
+  - package-ecosystem: "npm"
+    # Look for `package.json` and `lock` files in the `root` directory
+    directory: "/"
+    # Check the npm registry for updates every day (weekdays)
+    schedule:
+      interval: "weekly"
+
+  # Enable version updates for Docker
+  - package-ecosystem: "docker"
```
Comment (dhrubabasu) [inline .github/dependabot.yml:12]: Are we using docker in this repo?

File: .github/workflows/njsscan-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '28 3 * * 6'
+
+jobs:
+  njsscan:
```
Comment (dhrubabasu) [inline .github/workflows/njsscan-analysis.yml:16]: Not sure how useful njsscan will be, doesn't look like it has too much usage as well: https://github.com/ajinabraham/njsscan

**Review by dhrubabasu — CHANGES_REQUESTED** [review]
Left a few comments, LGTM overall

File: .github/dependabot.yml (hunk 1 above)
Comment (paOol) [reply .github/dependabot.yml]: we are not; removed this section.

File: .github/workflows/njsscan-analysis.yml (hunk 2 above)
Comment (paOol) [reply .github/workflows/njsscan-analysis.yml:16]: not sure, but we can easily remove later;
also added codeql in addition.

**Review by cgcardona — APPROVED** [review]
Looks good. TY! 🎩

**Pull request merged**